### DISCUSSION TITLE
Support for SQL syntaxes

### DISCUSSION THREAD

**Discussion by XufengXufengXufeng** [sqlalchemy/sqlalchemy#10045] [Usage Questions]
### Describe the use case

why can't i use sql syntax in 2.0 anymore? where is engine.execute? 

### Databases / Backends / Drivers targeted

sqlite3 python

### Example Use

engine.execute(sql)

**Comment by CaselIT** [comment]
Hi,

You will need to use a connection. Change documented here https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#implicit-and-connectionless-execution-bound-metadata-removed
